### ISSUE TITLE
Screenboard width and height should stay as type string

### ISSUE THREAD

**Issue by masci** [zorkian/go-datadog-api#192]
Hi all,

#190 changed the type for `Screenboard.Height` and `Screenboard.Width` from `*string` to `*int` (see https://github.com/zorkian/go-datadog-api/pull/190/files#diff-7d71d2abb56836dce85c7c7b114c80dcL20) but I believe they should stay as strings, at least `width` since it can be something like `"100%"`.

**Comment by nyanshak** [comment]
I agree that `*int` is not the right type, but I think `*string` is also not the right type. I think `*json.Number` is appropriate as it will accept strings such as `"1"`, `"100%"`, but also numbers such as `1` or `1.1`. Would you mind updating this PR to use `*json.Number` instead?

**Comment by mrwonko** [comment]
So it looks like this was supposed to be fixed by #195, which in turn was supposed to be based on #198, both of which are now closed without further explanation? I'll create a new pull request combining those two closed requests.
